src/containers/profile/settingsTab/styles.module.scss

```diff
@@ -0,0 +1,94 @@
+@import '../../../scss/shared/breakpoints.scss';
+@import '../../../scss/shared/fonts.scss';
+@import '../../../scss/shared/colors.scss';
+@import '../../../scss/mixins/button.scss';
+@import '../../../scss/mixins/input.scss';
+
```

Comment:
In my opinion, The best way for minify code is to move reiterative code in one file and then you can import this code by one row. Now you import it each time by five rows.